src/Summoner/Source.hs

```diff
@@ -1,15 +1,18 @@
+{-# LANGUAGE QuasiQuotes     #-}
+
 module Summoner.Source
        ( Source (..)
        , sourceT
        , fetchSource
        ) where
 
-import Control.Arrow ((>>>))
-import Control.Exception (catch)
-import System.Process (readProcess)
-import Toml (BiMap, BiToml, Key)
+import           Control.Arrow ((>>>))
+import           Control.Exception (catch)
+import           NeatInterpolation (text)
+import           System.Process (readProcess)
+import           Toml (BiMap, BiToml, Key)
 
-import Summoner.Ansi (errorMessage)
+import           Summoner.Ansi (errorMessage)
```

Comment:
Oops, I think these `stylish-haskell` changes are not intentional 🙂 
(Btw, we have `.stylish-haskell.yaml` in the repo with the config that we are using).